tools/runme/build.sh

```diff
@@ -109,13 +109,17 @@ _sbt_build() {
     TESTS=",$TESTS,"; TESTS="${TESTS//,+scala,/,}"; TESTS="${TESTS#,}"; TESTS="${TESTS%,}"
     if [[ "$TESTS" = "" ]]; then TESTS="all"; fi
   fi
-  _sbt_run "full-build"
-  show section "Sorting assembly jar for the maven repo"
-  # leave only the -assembley jars under the proper name (and the pom files)
-  local f; for f in "$BUILD_ARTIFACTS/packages/m2/"**; do case "$f" in
-    ( *-@(javadoc|sources).jar@(|.md5|.sha1) ) _rm "$f" ;;
-    ( *-assembly.jar@(|.md5|.sha1) ) _ mv "$f" "${f//-assembly.jar/.jar}" ;;
-  esac; done
+  if [[ "$SUBPROJECT" != "" ]]; then 
```

Comment:
@flybywind sorry, one question, why was this change necessary in build.sh?

Reply:
@flybywind I'm not sure if this specific change is necessary in this file, can it be removed?

Reply:
i have reverted the change and leave the problem to @mhamilton723